src/Raven.Client/Documents/Commands/UpdateSubscriptionCommand.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Session;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Commands
+{
+    public class UpdateSubscriptionCommand : RavenCommand<CreateSubscriptionResult>, IRaftCommand
+    {
+        private readonly SubscriptionCreationOptions _options;
+        private readonly long? _id;
+
+        public UpdateSubscriptionCommand(SubscriptionCreationOptions options, long? id = null)
+        {
+            _options = options;
+            _id = id;
```

Comment:
I do not think that we want that? All other operations are by Name?

Reply:
I added it because of those reasons
- support updating subscription name
- it is supported to create subscription by id:
https://github.com/ravendb/ravendb/blob/7750eec880a0f5b185d11fce76ff74e1d3530fa5/src/Raven.Client/Documents/Commands/CreateSubscriptionCommand.cs#L20